content/blog/2020/05/2020-05-11-docker-windows-agents.adoc

```diff
@@ -45,9 +45,9 @@ The jenkins/agent image is a simple agent with the JDK and the agent.jar (Jenkin
 There are two main use cases for this image:
 
  1. As a base image for other Docker images (e.g., `FROM jenkins/agent:jdk8-nanoserver-1809` in your Dockerfile). The jenkins/inbound-agent is based on this image.
- 2. This image may also be used to launch an agent using the *Launch method* of *Launch agent via execution of command on the master*.  This allows the master to launch the agent inside the docker container automatically.
+ 2. This image may also be used to launch an agent using the *Launch method* of *Launch agent via execution of command on the controller*.  This allows the controller to launch the agent inside the docker container automatically.
```

Comment:
This is referencing a UI label, so changing this post content before the plugin is changed makes no sense.

Reply:
I'm not sure I agree. The terminology is still changing and the blog post is using the old terminology. These changes are not about the plugins themselves, since those are not changing but the wording used in this blog post is not about the plugin so I feel these changes in these. instances are warranted

Reply:
"Launch agent via execution of command on the master" is a UI label, see screenshot on https://plugins.jenkins.io/command-launcher/

Reply:
I will correct this as well